src/daft-core/src/series/ops/agg.rs

```diff
@@ -97,6 +97,14 @@ impl Series {
         self.inner.max(groups)
     }
 
+    pub fn any_value(
```

Comment:
Zooming out a bit, we shouldn't have to implement this for each for the Arrays separately since this operation only has to work on the validity mask.
You could actually implement it on the series directly.
```
let mask = self.validity();
let indices = get_idx_from_bitmap(mask, groups);
self.take(indices)
```
I think count could also be refactored that way.
What do you think?

Reply:
I implemented this idea for any_value, but the array-level count implementation is used directly in parts of the code (eg. for computing mean) so I think it's still useful to consolidate the logic there